### PR TITLE
Add WEMIX Mainnet and Modify Testnet 

### PR DESCRIPTION
**Pull request by ReleaseWemix**
WEMIX3.0 Mainnet will be available at 2022-10-20 03:00:00 UTC

### CODE REVIEW

File: _data/chains/eip155-1111.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://wemix.com",
+  "shortName": "wemix",
+  "chainId": 1111,
+  "networkId": 1111,
+  "explorers": [
+    {
+      "name": "WEMIX Block Explorer",
+      "url": "https://explorer.wemix.com",
```
Comment (ligi) [inline _data/chains/eip155-1111.json:18]: this site seems to be down
image

**Review by ligi — CHANGES_REQUESTED** [review]

**Comment by bossmade806** [comment]
Merge pull requests

File: _data/chains/eip155-1111.json (hunk 1 above)
Comment (wm-jsong1230) [reply _data/chains/eip155-1111.json:18]: Yes, it will be ready Oct 20th 3 AM UTC. Thanks.

Comment (ligi) [reply _data/chains/eip155-1111.json:18]: Please ping me then or remove the explorer for now - currently I cannot verify EIP3091 compatibilty and so can't merge

Comment (wm-jsong1230) [reply _data/chains/eip155-1111.json:18]: Sure, I will ping you when it is up and ready to check. Thanks.

**Comment by mohammadzahoor** [comment]
tools/schemaCheck.js

File: _data/chains/eip155-1111.json (hunk 1 above)
Comment (ReleaseWemix) [reply _data/chains/eip155-1111.json:18]: Explorer is ready to use. Please confirm this.

Comment (wm-jsong1230) [reply _data/chains/eip155-1111.json:18]: @ligi 

> Please ping me then or remove the explorer for now - currently I cannot verify EIP3091 compatibilty and so can't merge

could you check this? Wemix explorer is ready to use (https://explorer.wemix.com)

Comment (wm-jsong1230) [reply _data/chains/eip155-1111.json:18]: @ligi could you please confirm?

**Review by ligi — APPROVED** [review]

**Comment by mohammadzahoor** [comment]
{"collectibles":{"favorites":{}},"engine":{"backgroundState":{"AccountTrackerController":{"accounts":{"0xb2766b84519fF2C7517DDF1BF0Ae45c53aFb649A":{"balance":"0x0"},"0x0fe2808A5F823C1406788cC68Ad4774F825F6De0":{"balance":"0x0"}},"_U":0,"_V":0,"_W":null,"_X":null},"AddressBookController":{"addressBook":{"1":{"0x87a70E33E2F690af84d7A3695a3C7AA581fF708c":{"address":"0x87a70E33E2F690af84d7A3695a3C7AA581fF708c","chainId":"1","isEns":false,"memo":"","name":"0x87a70e33e2f690af84d7a3695a3c7aa581ff708c","isSmartContract":false}}}},"TokenListController":{"tokenList":{},"tokensChainsCache":{}},"CurrencyRateController":{"conversionDate":1666023324.303,"conversionRate":0.007427,"nativeCurrency":"EXP","currentCurrency":"usd","pendingCurrentCurrency":null,"pendingNativeCurrency":null,"usdConversionRate":0.007427},"KeyringController":{"keyrings":[{"accounts":["0xb2766b84519fF2C7517DDF1BF0Ae45c53aFb649A","0x0fe2808A5F823C1406788cC68Ad4774F825F6De0"],"index":0,"type":"HD Key Tree"},{"accounts":[],"index":1,"type":"QR Hardware Wallet Device"}]},"PersonalMessageManager":{"unapprovedMessages":{},"unapprovedMessagesCount":0},"NetworkController":{"network":"1","isCustomNetwork":true,"provider":{"type":"rpc","ticker":"EXP","chainId":"2","rpcTarget":"https://node.expanse.tech","nickname":"Expanse Network"},"properties":{}},"PreferencesController":{"featureFlags":{},"frequentRpcList":[{"rpcUrl":"https://api.avax.network/ext/bc/C/rpc","chainId":"43114","ticker":"AVAX","nickname":"Avalanche Mainnet C-Chain","rpcPrefs":{"blockExplorerUrl":"https://snowtrace.io"}},{"rpcUrl":"https://arbitrum-mainnet.infura.io/v3/d039103314584a379e33c21fbe89b6cb","chainId":"42161","ticker":"ETH","nickname":"Arbitrum One","rpcPrefs":{"blockExplorerUrl":"https://explorer.arbitrum.io"}},{"rpcUrl":"https://rpc.ftm.tools/","chainId":"250","ticker":"FTM","nickname":"Fantom Opera","rpcPrefs":{"blockExplorerUrl":"https://ftmscan.com"}},{"rpcUrl":"https://bsc-dataseed1.binance.org/","chainId":"56","ticker":"BNB","nickname":"BNB Smart Chain","rpcPrefs":{"blockExplorerUrl":"https://bscscan.com"}},{"rpcUrl":"https://polygon-mainnet.infura.io/v3/d039103314584a379e33c21fbe89b6cb","chainId":"137","ticker":"MATIC","nickname":"Polygon Mainnet","rpcPrefs":{"blockExplorerUrl":"https://polygonscan.com"}},{"rpcUrl":"https://api.harmony.one/","chainId":"1666600000","ticker":"ONE","nickname":"Harmony Mainnet Shard 0","rpcPrefs":{"blockExplorerUrl":"https://explorer.harmony.one"}},{"rpcUrl":"https://optimism-mainnet.infura.io/v3/d039103314584a379e33c21fbe89b6cb","chainId":"10","ticker":"ETH","nickname":"Optimism","rpcPrefs":{"blockExplorerUrl":"https://optimistic.etherscan.io"}},{"rpcUrl":"https://palm-mainnet.infura.io/v3/d039103314584a379e33c21fbe89b6cb","chainId":"11297108109","ticker":"PALM","nickname":"Palm","rpcPrefs":{"blockExplorerUrl":"https://explorer.palm.io"}},{"rpcUrl":"https://mainnet.optimism.io/","chainId":"10","ticker":"ETH","nickname":"ETHEREUM","rpcPrefs":{"blockExplorerUrl":"http://optimism.etherscan.io"}},{"rpcUrl":"https://node.expanse.tech","chainId":"2","ticker":"EXP","nickname":"Expanse Network","rpcPrefs":{"blockExplorerUrl":null}},{"rpcUrl":"https://www.ethercluster.com/etc","chainId":"61","ticker":"ETC","nickname":"Ethereum Classic Mainnet","rpcPrefs":{"blockExplorerUrl":"https://blockscout.com/etc/mainnet"}},{"rpcUrl":"https://rpc.ntity.io","chainId":"197710212030","ticker":"NTT","nickname":"Ntity Mainnet","rpcPrefs":{"blockExplorerUrl":"https://blockscout.ntity.io"}},{"rpcUrl":"https://frankenstein-rpc.oneledger.network","chainId":"4216137055","ticker":"OLT","nickname":"OneLedger Testnet Frankenstein","rpcPrefs":{"blockExplorerUrl":"https://frankenstein-explorer.oneledger.network"}},{"rpcUrl":"https://api.s1.b.hmny.io","chainId":"1666700001","ticker":"ONE","nickname":"Harmony Testnet Shard 1","rpcPrefs":{"blockExplorerUrl":null}}],"identities":{"0xb2766b84519fF2C7517DDF1BF0Ae45c53aFb649A":{"address":"0xb2766b84519fF2C7517DDF1BF0Ae45c53aFb649A","name":"Account 1","importTime":1654618624273},"0x0fe2808A5F823C1406788cC68Ad4774F825F6De0":{"address":"0x0fe2808A5F823C1406788cC68Ad4774F825F6De0","name":"Zahoornoor","importTime":1659215286646}},"ipfsGateway":"https://cloudflare-ipfs.com/ipfs/","lostIdentities":{},"selectedAddress":"0xb2766b84519fF2C7517DDF1BF0Ae45c53aFb649A","useStaticTokenList":false,"useCollectibleDetection":true,"openSeaEnabled":true,"_U":0,"_V":0,"_W":null,"_X":null},"TokenBalancesController":{"contractBalances":{}},"TokenRatesController":{"contractExchangeRates":{}},"TransactionController":{"methodData":{},"transactions":[],"internalTransactions":[],"swapsTransactions":{}},"TypedMessageManager":{"unapprovedMessages":{},"unapprovedMessagesCount":0},"SwapsController":{"quotes":{},"quoteValues":{},"fetchParams":{"slippage":0,"sourceToken":"","sourceAmount":0,"destinationToken":"","walletAddress":""},"fetchParamsMetaData":{"sourceTokenInfo":{"decimals":0,"address":"","symbol":""},"destinationTokenInfo":{"decimals":0,"address":"","symbol":""}},"topAggSavings":null,"aggregatorMetadata":null,"tokens":null,"topAssets":null,"approvalTransaction":null,"aggregatorMetadataLastFetched":0,"quotesLastFetched":0,"topAssetsLastFetched":0,"error":{"key":null,"description":null},"topAggId":null,"tokensLastFetched":0,"isInPolling":false,"pollingCyclesLeft":3,"quoteRefreshSeconds":null,"usedGasEstimate":null,"usedCustomGas":null,"chainCache":{"1":{"aggregatorMetadata":null,"tokens":null,"topAssets":null,"aggregatorMetadataLastFetched":0,"topAssetsLastFetched":0,"tokensLastFetched":0}}},"GasFeeController":{"gasFeeEstimates":{},"estimatedGasFeeTimeBounds":{},"gasEstimateType":"none"}}},"privacy":{"privacyMode":true,"thirdPartyApiMode":true,"approvedHosts":{"looksrare.org":true,"chat.blockscan.com":true,"chainlist.wtf":true,"app.ens.domains":true}},"bookmarks":[],"recents":["0x0fe2808A5F823C1406788cC68Ad4774F825F6De0","0x87a70E33E2F690af84d7A3695a3C7AA581fF708c","0x3Bb974669829CA6046Ccb53868C553c280ca2f20"],"browser":{"activeTab":1666023372766,"history":[{"url":"https://opensea.io/","name":"https://opensea.io"},{"url":"https://opensea.io/","name":"https://opensea.io"},{"url":"https://opensea.io/","name":"Access denied"},{"url":"https://opensea.io/","name":"https://opensea.io"},{"url":"https://opensea.io/","name":"https://opensea.io"},{"url":"https://opensea.io/","name":"Access denied"},{"url":"https://opensea.io/","name":"https://opensea.io"},{"url":"https://opensea.io/","name":"https://opensea.io"},{"url":"https://opensea.io/","name":"Access denied"},{"url":"https://home.metamask.io/","name":"https://home.metamask.io"},{"url":"https://home.metamask.io/","name":"https://home.metamask.io"},{"url":"https://home.metamask.io/","name":"MetaMask Home"},{"url":"https://www.cloudflare.com/","name":"https://www.cloudflare.com"},{"url":"https://www.cloudflare.com/","name":"https://www.cloudflare.com"},{"url":"https://www.cloudflare.com/","name":"Cloudflare - The Web Performance & Security Company | Cloudflare"},{"url":"https://events.www.cloudflare.com/flow/cloudflare/connect22london/landing/page/page","name":"https://events.www.cloudflare.com/flow/cloudflare/connect22london/landing/page/page"},{"url":"https://events.www.cloudflare.com/flow/cloudflare/connect22london/landing/page/page","name":"https://events.www.cloudflare.com/flow/cloudflare/connect22london/landing/page/page"},{"url":"https://events.www.cloudflare.com/flow/cloudflare/connect22london/landing/page/page","name":"https://events.www.cloudflare.com/flow/cloudflare/connect22london/landing/page/page"},{"url":"https://events.www.cloudflare.com/flow/cloudflare/connect22london/landing/page/page","name":"https://events.www.cloudflare.com/flow/cloudflare/connect22london/landing/page/page"},{"url":"https://opensea.io/","name":"https://opensea.io"},{"url":"https://opensea.io/","name":"https://opensea.io"},{"url":"https://opensea.io/","name":"Access denied"},{"url":"https://events.www.cloudflare.com/flow/cloudflare/connect22london/landing/page/page","name":"Connect 2022 London"},{"url":"https://looksrare.org/","name":"https://looksrare.org"},{"url":"https://looksrare.org/","name":"https://looksrare.org"},{"url":"https://looksrare.org/","name":"LooksRare - NFT Marketplace | LooksRare"},{"url":"https://looksrare.org/rewards","name":"Rewards | LooksRare"},{"url":"https://looksrare.org/rewards","name":"Rewards | LooksRare"},{"url":"https://looksrare.org/rewards","name":"Rewards | LooksRare"},{"url":"https://looksrare.org/rewards","name":"https://looksrare.org/rewards"},{"url":"https://looksrare.org/rewards","name":"https://looksrare.org/rewards"},{"url":"https://events.www.cloudflare.com/flow/cloudflare/connect22london/landing/page/page","name":"https://events.www.cloudflare.com/flow/cloudflare/connect22london/landing/page/page"},{"url":"https://events.www.cloudflare.com/flow/cloudflare/connect22london/landing/page/page","name":"https://events.www.cloudflare.com/flow/cloudflare/connect22london/landing/page/page"},{"url":"https://looksrare.org/rewards","name":"Rewards | LooksRare"},{"url":"https://events.www.cloudflare.com/flow/cloudflare/connect22london/landing/page/page","name":"Connect 2022 London"},{"url":"https://home.metamask.io/","name":"https://home.metamask.io"},{"url":"https://home.metamask.io/","name":"https://home.metamask.io"},{"url":"https://home.metamask.io/","name":"MetaMask Home"},{"url":"https://chat.blockscan.com/connect-wallet","name":"https://chat.blockscan.com/connect-wallet"},{"url":"https://chat.blockscan.com/connect-wallet","name":"https://chat.blockscan.com/connect-wallet"},{"url":"https://chat.blockscan.com/connect-wallet","name":"Blockscan Chat - Wallet to Wallet Messaging for the Web3 Blockchain"},{"url":"https://chat.blockscan.com/connect-wallet#","name":"Blockscan Chat - Wallet to Wallet Messaging for the Web3 Blockchain"},{"url":"https://chat.blockscan.com/connect-wallet#","name":"Blockscan Chat - Wallet to Wallet Messaging for the Web3 Blockchain"},{"url":"https://chat.blockscan.com/connect-wallet","name":"https://chat.blockscan.com/connect-wallet"},{"url":"https://chat.blockscan.com/connect-wallet","name":"https://chat.blockscan.com/connect-wallet"},{"url":"https://chat.blockscan.com/connect-wallet","name":"Blockscan Chat - Wallet to Wallet Messaging for the Web3 Blockchain"},{"url":"https://home.metamask.io/","name":"https://home.metamask.io"},{"url":"https://home.metamask.io/","name":"https://home.metamask.io"},{"url":"https://home.metamask.io/","name":"MetaMask Home"},{"url":"https://etherscan.io/address/0x71c7656ec7ab88b098defb751b7401b5f6d8976f","name":"https://etherscan.io/address/0x71c7656ec7ab88b098defb751b7401b5f6d8976f"}],"whitelist":[],"tabs":[{"url":"https://app.ens.domains/","id":1661729010023,"image":"file:///data/user/0/io.metamask/cache/ReactNative-snapshot-image1006984611276114607.jpg"},{"url":"https://chat.blockscan.com/connect-wallet#","id":1661729041485,"image":"file:///data/user/0/io.metamask/cache/ReactNative-snapshot-image1513200165896618363.jpg"},{"url":"https://etherscan.io/login","id":1661729139661,"image":"file:///data/user/0/io.metamask/cache/ReactNative-snapshot-image8012292168182767605.jpg"},{"url":"https://accounts.google.com/ServiceLogin/webreauth?continue=https%3A%2F%2Fpolicies.google.com%2Ftechnologies%2Fpartner-sites&dsh=S-1274434703%3A1665241704914423&flowEntry=ServiceLogin&flowName=GlifWebSignIn&followup=https%3A%2F%2Fpolicies.google.com%2Ftechnologies%2Fpartner-sites&ifkv=AQDHYWowop8ZvpGVSi9cJHU_HlNxvVp5RXdOvcuUlKlvw-BZ8h5k65qBXi2q0V0KJXZIA1KEStyVXQ","id":1661730600132,"image":"file:///data/user/0/io.metamask/cache/ReactNative-snapshot-image4515023582599064617.jpg"},{"url":"https://metamask.io/download/","id":1664504179780},{"url":"intent://open?_branch_referrer=H4sIAAAAAAAAA8soKSkottLXz00tScxNLM7WSywo0MvJzMvWT6ooz3a1yHeztLSPj08qSsxLzohPzC2IzyrOz7M1VIMJ5SaWAMnMFFtDQwMTI0NzCwNTU0NDCyMgx0CtKDUttagotcgWZGJ8ck5mcjZQqaqRqbELFuUA%2FNNA%2FIwAAAA%3D&link_click_id=1073427048975176884#Intent;scheme=metamask;package=io.metamask;S.browser_fallback_url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dio.metamask&referrer=link_click_id%253D1073427048975176884;S.market_referrer=link_click_id%3D1073427048975176884&__branch_flow_type%3Dviewapp%26__branch_flow_id%3D1104217805520209029%26__branch_mobile_deepview_type%3D1%26__branch_amp_json%3D1%26_branch_match_id%3D1104217805511820420%26referrer%3Dlink_click_id%253D1104217805511820420%26_t%3D1104217805511820420;S.branch_data=%7B%22%24og_title%22%3A%22Introducing%20MetaMask%20Mobile%22%2C%22%24marketing_title%22%3A%22Website%20-%20Download%20CTA%20-%20Android%22%2C%22~creation_source%22%3A1%2C%22%24og_description%22%3A%22More%20than%20a%20wallet.%20%7C%20Now%20Available%20on%20iOS%20and%20Android%22%2C%22%24og_image_url%22%3A%22https%3A%2F%2Fmetamask.app.link%2FskAH3BaF99%22%2C%22~feature%22%3A%22marketing%22%2C%22_branch_match_id%22%3A%221104217805511820420%22%2C%22%2Burl%22%3A%22https%3A%2F%2Fmetamask.app.link%2FbxwkE8oF99%3F__branch_amp_json%3D1%26_branch_match_id%3D1104217805511820420%26referrer%3Dlink_click_id%25253D1104217805511820420%22%2C%22referrer%22%3A%22link_click_id%253D1104217805511820420%22%2C%22%24desktop_url%22%3A%22https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dio.metamask%26hl%3Den_US%26ref%3Dproducthunt%22%2C%22~marketing%22%3Atrue%2C%22~id%22%3A%22839191046432575227%22%2C%22%24one_time_use%22%3Afalse%2C%22__branch_amp_json%22%3A%221%22%2C%22%2Bdomain%22%3A%22metamask.app.link%22%2C%22%2Bclick_timestamp%22%3A1664504266%2C%22%2Breferrer%22%3A%22io.metamask%22%2C%22%2Bclicked_branch_link%22%3Atrue%2C%22%2Bmatch_guaranteed%22%3Atrue%2C%22%2Bis_first_session%22%3Afalse%7D;B.branch_intent=true;end","id":1664504220248,"image":"file:///data/user/0/io.metamask/cache/ReactNative-snapshot-image8221477225500978369.jpg"},{"url":"https://chainlist.wtf/","id":1666023372766}]},"modals":{"networkModalVisible":false,"accountsModalVisible":false,"collectibleContractModalVisible":false,"receiveModalVisible":false,"dappTransactionModalVisible":false,"approveModalVisible":false},"settings":{"useBlockieIcon":true,"hideZeroBalanceTokens":true,"searchEngine":"Google","primaryCurrency":"Fiat","lockTime":30000,"showHexData":true,"showCustomNonce":true},"alert":{"isVisible":false,"autodismiss":null,"content":"clipboard-alert","data":{"msg":"Public address copied to clipboard"}},"transaction":{"selectedAsset":{},"transaction":{}},"user":{"loadingMsg":"","loadingSet":false,"backUpSeedphraseVisible":false,"protectWalletModalVisible":false,"nftDetectionDismissed":false,"isAuthChecked":true,"initialScreen":"login","appTheme":"os","userLoggedIn":true,"gasEducationCarouselSeen":true,"passwordSet":true,"seedphraseBackedUp":true},"wizard":{"step":0},"onboarding":{"events":[]},"notification":{"notifications":[]},"swaps":{"1":{"isLive":true},"56":{"isLive":false},"43114":{"isLive":true},"isLive":true,"hasOnboarded":true},"fiatOrders":{"selectedCountry":"US","selectedPaymentMethodAgg":"/payments/debit-credit-card","selectedRegionAgg":{"emoji":"ðŸ‡ºðŸ‡¸","id":"/regions/us-ak","name":"Alaska","unsupported":false},"getStartedAgg":true,"orders":[]},"infuraAvailability":{"isBlocked":false},"navigation":{"currentRoute":"AdvancedSettings"},"networkOnboarded":{"networkOnboardedState":[{"network":"https://node.expanse.tech","onboarded":true},{"network":"https://api.s1.b.hmny.io","onboarded":true},{"network":"https://arbitrum-mainnet.infura.io/v3/d039103314584a379e33c21fbe89b6cb","onboarded":true},{"network":"https://rpc.ftm.tools","onboarded":true},{"network":"https://rpc.ftm.tools","onboarded":true},{"network":"https://mainnet.optimism.io","onboarded":true},{"network":"goerli","onboarded":true},{"network":"ropsten","onboarded":true},{"network":"mainnet","onboarded":true},{"network":"rinkeby","onboarded":true}],"networkState":{"showNetworkOnboarding":false,"nativeToken":"","networkType":"","networkUrl":""},"switchedNetwork":{"networkUrl":"","networkStatus":false}},"_persist":{"version":10,"rehydrated":true}}

**Comment by joeleonjr** [comment]
Hi! I'm a security researcher @TruffleSecurity. I found a valid API key leaked in a comment in this Issue. I highly recommend rotating the API key immediately (not just deleting the comment). Here's a blog post I wrote on remediating these types of vulnerabilities: https://trufflesecurity.com/blog/remediate-leaked-api-keys-with-key-rotation.<br><br>If you can't find the leaked API key, you can use the following [TruffleHog](https://github.com/trufflesecurity/trufflehog) command to automatically scan for it:<br>`trufflehog github --repo https://github.com/<your_username>/<your_reponame> --issue-comments --only-verified`<br><br>If it's not clear where/what the security issue is, please let me know. I'd be happy to help. Please note that the API key might be in the past edit of a comment.<br><br><br><br>(cc @mohammadzahoor)

**Pull request merged**